design-documents/catalog-domain/understanding-of-complex-product-types.md

```diff
@@ -0,0 +1,33 @@
+# Understanding of complex product types
+
+## 1. Definitions
+See [Simple Products](simple-products.md)
+* Complex product is a virtual representation of multiple real SKUs at the storefront.
+* Currently, Magento does support three complex product types - Bundle, Grouped and Configurable.
+* For the simplicity and data consistency a complex product has SKU.
+In fact this SKU is virtual.
+* Complex product may have all attributes that required to render product details page. 
+* Complex product does not have attributes that describe physical product characteristics.
```

Comment:
What is considered "physical product characteristics"? Or could you add some examples? For example, is inventory included or not? If not, are there scenarios where admin would like to limit number of sold complex products? If yes, would it be a different characteristic (not inventory) or what is proposed?